src/pkg/services/m365/api/lists.go

```diff
@@ -62,3 +63,228 @@ func (c Lists) PostDrive(
 
 	return drive, graph.Wrap(ctx, err, "fetching created documentLibrary").OrNil()
 }
+
+// SharePoint lists represent lists on a site. Inherits additional properties from
```

Comment:
It's difficult to identify changes when code blocks get moved.  Preference is to have those movements occur in a  separate PR so that changes are highlighted.  Could you please make a PR that separates out this movement from the rest of the changes?